doc/whats_new/v1.2.rst

```diff
@@ -38,6 +38,10 @@ random sampling procedures.
   It is now consistent and deterministic with all SVD solvers.
   :pr:`22527` by :user:`Meekail Zain <micky774>` and `Thomas Fan`_.
 
+- |Fix| :class:`feature_extraction.image.PatchExtractor` required to be fitted
```

Comment:
Do we use `|API|` for this because users need to change their code to have the same functionality?